src/lib.rs

```diff
@@ -25,10 +25,10 @@ pub mod crds_gossip_push;
 pub mod crds_value;
 #[macro_use]
 pub mod contact_info;
-pub mod blockstream;
-pub mod blockstream_service;
 #[macro_use]
 pub mod blocktree;
+pub mod blockstream;
```

Comment:
@CriesofCarrots this was the problem. macros need to be defined before used

Reply:
Yeah, only if the macro is defined within the same crate. Rust fail! That tripped me up for hours when I moved counter.rs into `solana_metrics`.

Reply:
Ohhh, thank you!!